src/modules/nav/index.tsx

```diff
@@ -98,7 +99,19 @@ export const Navbar = () => {
               />
             </NextLink>
           )}
-
+          <NextLink href={"https://github.com/polygon-isecure/"} passHref>
+            <Button style={{backgroundColor: 'transparent', transition: '0.24s ease'}} _hover={{
+              transform: 'scale(1.1)',
+              cursor: 'pointer',
+              }}
+              >
+              <GitHub style={{
+                color: 'white',
+                borderRadius: '50px',
+                padding: '8px',
+              }} size="3rem" />
+            </Button>
+          </NextLink>
```

Comment:
Not too sure about this one to be honest... There is already a GitHub button in the footer section redirecting to our GitHub page.

Reply:
I mean, I think a lot of people wouldn't scroll all the way down to the footer. It's personal preference though

Reply:
I see. In that case, I think it would be nice to set `spacing={1}` for the parent `Stack` component.

Reply:
Done